### PR TITLE
FEAT: Stream ChatTTS

### PR DESCRIPTION
**Pull request by codingl2k1**
- Add an additional `stream` option to the xinference_client API, default is False.
- The openai API use non stream mode by default as the ChatTTS stream may not work as well as non stream.
- The default stream value can be changed by an environment var: `export XINFERENCE_AUDIO_SPEECH_DEFAULT_STREAM=1`
- Stream mode requires torchaudio + ffmpeg<7 (Can be installed by `conda install -c conda-forge 'ffmpeg<7'`), please refer to: https://pytorch.org/audio/stable/installation.html

### CODE REVIEW

**Comment by ChengjieLi28** [comment]
Please update chattts dependency in docker requirements file: `xinference/deploy/docker/requirements.txt`

File: xinference/client/restful/restful_client.py [hunk 1]
```diff
@@ -692,6 +692,7 @@ def speech(
         voice: str = "",
         response_format: str = "mp3",
         speed: float = 1.0,
+        stream: bool = False,
```
Comment (qinxuye) [inline xinference/client/restful/restful_client.py:695]: The environment variable can be set, but here it is by default false.

**Comment by qinxuye** [comment]
Shall us do `conda install -c conda-forge 'ffmpeg<7'` in the dockerfile?

File: xinference/deploy/docker/requirements.txt [hunk 2]
```diff
@@ -46,11 +46,11 @@ timm>=0.9.16  # For deepseek VL
 torchvision  # For deepseek VL
 FlagEmbedding  # For rerank
 omegaconf~=2.3.0  # For ChatTTS
-vector_quantize_pytorch  # For ChatTTS
-vocos  # For ChatTTS
 nemo_text_processing  # For ChatTTS
 WeTextProcessing  # For ChatTTS
 librosa  # For ChatTTS
+torchaudio  # For ChatTTS
+ChatTTS>0.1
```
Comment (qinxuye) [inline xinference/deploy/docker/requirements.txt:53]: seems this is absent in `requirements_cpu.txt'

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**